config/config.go

```diff
@@ -77,10 +78,11 @@ var (
 	debug    = flag.Bool("debug", false, "set debug log verbosity")
 	stdout   = flag.Bool("stdout", false, "log to stdout")
 
-	agentConfig   = &config{}
-	agentConfigMx sync.RWMutex
-	version       string
-	lEtag         = &lastEtag{Etag: "0"}
+	agentConfig      = &config{}
+	lAgentConfigHash = ""
```

Comment:
minor: why `l`? Is it for `last`? Isn't this the hash of the `agentConfig`?

Reply:
right. `l` is for last. the prefix is consistent with other variables that has been used to cache previous values, for example `lEtag`

Reply:
ok.. i see your concern around not being consistent with `agentConfig`. `lAgentConfigHash` does not sound like it is the hash `agentConfig`.

I will go ahead and change the variable name to `agentConfigHash`